integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -1001,7 +1001,7 @@ void testDropCatalogAndCreateAgain() throws InterruptedException {
       createdMetalake.dropCatalog(NameIdentifier.of(metalakeName, catalogName));
       // We need to test we can't load this catalog any more by Trino.
       success = checkTrinoHasRemoved(sql, 30);
-      Assertions.assertFalse(success, "Trino should not load the catalog any more: " + sql);
+      Assertions.assertTrue(success, "Trino should not load the catalog any more: " + sql);
```

Comment:
Can you add some description about why the Trino image `0.1.3` brings these changes?